tests/functional/test_libraries.py

```diff
@@ -365,6 +365,15 @@ def test_keyring(pyi_builder):
     pyi_builder.test_source("import keyring")
 
 
+@importorskip('keyring')
+def test_keyring_get_password(pyi_builder):
+    pyi_builder.test_source(
+        """
+        import keyring
+        keyring.get_password("test", "test")
+        """)
```

Comment:
I suggest merging this into the other test-case, which is basically contained in this one for 100%. Test-cases are quite expensive (complete freeze plus spawning a process), thus we should avoid trivial tests (like to existing one) if we better onws (like the new one).

Reply:
OK, I've replaced the old test with the new extended one. The main motivation for keeping the original test was the fact that the new one needs to be skipped (or perhaps xfail'd) on linux. But I suppose the original test did not really cover the base usage, so it does not make much of a difference.